src/actions/link.actions.js

```diff
@@ -1,14 +1,49 @@
 import {
 	FLOWDESIGNER_LINK_ADD,
+	FLOWDESIGNER_LINK_UPDATE,
```

Comment:
notice: try to add hierarchy in those constants, having it all flat is a pain

Reply:
most of those will be deprecated anyway, max 4 actions by files will be kept